helpers/auth_x11.c

```diff
@@ -62,11 +63,14 @@ const char *authproto_executable;
 //! The maximum time to wait at a prompt for user input in seconds.
 int prompt_timeout;
 
+//! Number of dancers in the disco password display
+#define DISCO_PASSWORD_DANCERS 5
+
 //! Length of the "paranoid password display".
-#define PARANOID_PASSWORD_LENGTH 32
+#define PARANOID_PASSWORD_LENGTH (1 << DISCO_PASSWORD_DANCERS)
 
 //! Minimum distance the cursor shall move on keypress.
-#define PARANOID_PASSWORD_MIN_CHANGE 4
+#define PARANOID_PASSWORD_MIN_CHANGE 5
```

Comment:
Not objecting, but why?

Reply:
Accidentally changed - reverted in 9e72fea.